### PR TITLE
Remove ECCS

### PR DESCRIPTION
**Pull request by philip-cc**
### Description
Over time the ECCS sample application has basically been reduced to a thin CLI wrapper around the go client. Additionally, I don't think it really aligns with our new "slices" strategy. I propose removing it. 

### Type(s) of Change (Split the PR if you check many)
- [ ] Added user feature
- [ ] Added technical feature
- [ ] Added tests
- [x] Refactor
- [ ] Bugfix
- [ ] Updated documentation
- [ ] Kubernetes changes
- [ ] CI/CD workflow changes

### Testing Checklist
- [ ] I have added/updated unit tests for functional changes.
- [ ] I have added/updated integration tests for changes that affect dependent systems.
- [ ] I have added/updated end-to-end tests for feature changes.

### General Checklist
- [ ] I have pulled in the latest changes from master.
- [ ] I have run `make lint`.
- [ ] I have successfully run `make tests`.
- [x] I have updated relevant CI/CD workflows.
- [ ] I have updated relevant Kubernetes manifests/scripts.
- [x] I have updated/added relevant (internal and external) documentation (readme, doc comments, etc).
- [ ] I have updated the dependency map to reflect my changes.
- [ ] I have added the license to new source code files.
- [ ] I have spent some time looking over the full diff before creating this PR.

### Technical Debt
Updating the dependency map

### CODE REVIEW

**Comment by kim-cc** [comment]
Yes, it seems like a a very thin wrapper. As I don't know much about the application, I don't know if it makes sense to delete. But what I can say is that, it hardly seems like a sample anymore, it is such a thin wrapper that it doesn't show anything, other than calling the API directly. It doesn't really provide more knowledge, than looking at the documentation (which is being worked on). Instead I think a better sample should be created at some point, giving users more value than this thin wrapper.
Bottom line is I think deleting it would be a good idea

**Review by gottlieb76 — APPROVED** [review]
:+1:
